en/TripUpdate.md

```diff
@@ -11,6 +11,6 @@ General guidelines for trip cancellations:
 | vehicle | refer to [message VehicleDescriptor](#VehicleDescriptor) |
 |  | If separate `VehiclePosition` and `TripUpdate` feeds are provided, [TripDescriptor](#TripDescriptor) and [VehicleDescriptor](#VehicleDescriptor) ID values pairing should match between the two feeds.<br/>For example, a `VehiclePosition` entity has `vehicle_id:A` and `trip_id:4`, then the corresponding `TripUpdate` entity should also have `vehicle_id:A` and `trip_id:4`. If any `TripUpdate` entity has `trip_id:4` and any `vehicle_id` other than 4, this is an error. |
 | stop_time_update | `stop_time_updates` for a given `trip_id` should be strictly ordered by increasing `stop_sequence` and no `stop_sequence` should be repeated. |
-|  | All TripUpdates should include at least one `stop_time_update` with a predicted arrival or departure time in the future. If all `stop_time_updates` for a trip reference past arrival and departure times, consumers should assume that no real-time data is available for the trip. |
+|  | All TripUpdates should include at least one stop_time_update with a predicted arrival or departure time in the future (or recent past) while the trip is in progress (or scheduled to be in progress). |
```

Comment:
@paulswartz Coming back and looking at this with fresh eyes, I don't think the this new language is clear enough to avoid the case I mentioned in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/pull/15#discussion_r644859538 (where all stop_time_updates are consistently in the past).

Could we reword this as:

>While the trip is in progress, all TripUpdates should include at least one stop_time_update with a predicted arrival or departure time in the future. Note that the [GTFS Realtime spec](https://github.com/google/transit/blob/master/gtfs-realtime/spec/en/trip-updates.md#stop-time-updates) says that producers should not drop a past StopTimeUpdate if it refers to a stop with a scheduled arrival time in the future for the given trip (i.e. the vehicle has passed the stop ahead of schedule), as otherwise it will be concluded that there is no update for this stop.

Reply:
@barbeau I don't think that's quite right. In the case where a vehicle gets to the final stop early, consumers should continue to provide the TripUpdate until the *scheduled* times are in the past, not only the StopTimeUpdates.

Reply:
@paulswartz Ah, good point - and that language is copy/paste from the spec, so it should change too.

What if we remove the work "past" from "past StopTimeUpdate" (both here and in the spec)?

>While the trip is in progress, all TripUpdates should include at least one stop_time_update with a predicted arrival or departure time in the future. Note that the [GTFS Realtime spec](https://github.com/google/transit/blob/master/gtfs-realtime/spec/en/trip-updates.md#stop-time-updates) says that producers should not drop a StopTimeUpdate if it refers to a stop with a scheduled arrival time in the future for the given trip (i.e. the vehicle has passed the stop ahead of schedule), as otherwise it will be concluded that there is no update for this stop.